packages/components/src/text/types.ts

```diff
@@ -28,12 +28,7 @@ export interface Props extends TruncateProps {
 	/**
 	 * Automatically calculate the appropriate line-height value for contents that render text and Control elements (e.g. `TextInput`).
 	 */
-	adjustLineHeightForInnerControls?:
-		| boolean
```

Comment:
What's the reason for removing the boolean type here? As much as I also don't like mixing boolean & string enums, I'm afraid we can't change a component's public API

Reply:
I **entirely** misunderstood what you said in Slack.

Reply:
I'll take a look again to make sure I'm not the one completely forgetting the context around this change

Reply:
Ok, my apologies. You understood correctly, and I should probably take a break from my laptop from time to time 😂  I had completely forgotten about the context of this prop type.

Your initial changes were correct here. We can remove the `boolean` type from the types:

- passing `false` has the same behavior of passing `undefined`
- passing `true` has the same behavior of passing `'medium'`
- consumers that were passing a boolean won't notice any changes in how the component behaves (since we're not introducing runtime changes), apart from potential TS errors about the prop not supporting boolean values